src/components/Rooms/AddRoom.js

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
+import '../styles/AddRoom.css';
+
+function AddRoom() {
```

Comment:
- [ ] Kindly refactor this functional component to use ES6.   Reason:  The vast majority of the software industry has deprecated the`function` keyword.

That way, your app will comply with the latest standards 💪

Reply:
Thank you so much for the feedback, Miguel. I will handle this issue.